MessageKit.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
    s.name = 'MessageKit'
-   s.version = '3.1.0'
+   s.version = '3.1.3'
```

Comment:
Why not 3.1.1?

Reply:
I created 3.1.1, 3.1.2, 3.1.3 in my fork, so forgot to change, now I have updated to 3.1.1